TRANSLATE.rst

```diff
@@ -0,0 +1,52 @@
+******************
+Translation Guide 
+******************
+
+.. contents:: 
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages availables:
+
+- Spanish contributed by Leonardo J. Caballero G.
+- Your Language by YOUR NAME.
+
+Installation
+============
+
+For translate the Read the Docs Sphinx Theme you need install the following packages:
+
+.. code:: bash
+
+    $ pip install babel Jinja2
+
+Distutils/Setuptools Integration
+================================
+
+You can configure different parts of the theme.
+
+Compile catalog
+---------------
+
+For compile catalog of the theme's project-wide is available. 
+More options please, check out http://babel.pocoo.org/en/latest/setup.html#compile-catalog
+
+Extract messages
+----------------
+
+It is available the option to extract messages to translate. 
+More options please, check out http://babel.pocoo.org/en/latest/setup.html#extract-messages
+
+init catalog
+------------
+
+It is available the option to init catalog. 
+More options please, check out http://babel.pocoo.org/en/latest/setup.html#init-catalog
+
+update catalog
+--------------
+
+It is available the option to init catalog. 
+More options please, check out http://babel.pocoo.org/en/latest/setup.html#update-catalog
```

Comment:
This disutils section doesn't seem to clarify anything. It can either be dropped completely, or updated to be a list of explicit steps an author will need to take to create a translation.